data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1791 @@
+			choice
+				`	"Help with what, exactly?"`
+				`	"I'm sorry, but helping your group any further sounds too risky to me."`
+					decline
+			label help
+			`	"Difficult it is, to fight Heliarch ships, but fight them all the same we eventually will need to," she explains. "Not very... experienced in warfare, most of our members are, as avoiding any ship battles we have been. Seeing as a jump drive you have, thought we did that bring some of our regional fleet leaders to human space, you could. Train there against human pirates, we can."`
+			choice
+				`	"Alright. I'll help you. How many people are you bringing along?"`
+				`	"I'm sorry, but helping your group any further sounds too risky to me."`
+					decline
+			`	She perks up with your answer. "<bunks> in total, we are. Wish to cause a commotion in human space, we do not, so bringing along <cargo> to sustain ourselves for a while, we are. Avoid leaving the ships at all, we will."`
+				goto accept
+			label resistance
+			`	You recognize her as Pyakri, a Lunarium military leader who you met on Remote Blue, and bring her aboard your ship to talk. "Looking for you I have been, Captain <last>. Need your help I do, for a training endeavor of sorts."`
+			choice
+				`	"What do you need help with?"`
+					goto with
+				`	"I'm not very good at training others, are you sure you need my help?"`
+			`	Her mandibles click together rapidly, sounding almost like a chime in its rhythm. "Doing the training you will not be. For the most part, at least," she explains. "Waiting just outside, many of our regional fleet leaders are. Your jump drive, and some of your knowledge, what we need is." She chirps again.`
+			label with
+			`	"Very... lacking most of our members are, in ship to ship combat. Wish to test some tactics, they do, so that better explain it to members under them they can. Of course, dangerous to challenge the Heliarchs for that, it is, so thought of training against human pirates, we did. Bring us to human space for that, you would. <bunks> in total we are, and bring along <cargo> to sustain ourselves without needing to leave the ships and cause a commotion, we will."`
+			label accept
+			`	She asks you to open the hatch, and she has a few more members from the crowd come in. They ask for your map, and then for the nearest hub of pirate activity. You point to Shaula, home to Greenrock, the biggest pirate world in southern human space, telling them it's one of the most dangerous ones. They then ask if there are any major shipyards in the South, and you point them to Southbound Shipyards, on <planet>, and the Tarazed Corporation, on Wayfarer. After some debate, they decide that <planet>'s proximity to the Coalition makes it a better pick in case they need to return quickly, and settle on that as the destination. "Bringing along enough credits to purchase a few warships, I hope we are," Pyakri comments. They have their <cargo> brought to your ship, and you show them to their bunks.`
```

Comment:
```suggestion
			`	She asks you to open the hatch, and she has a few more members from the crowd come in. They ask for your map, and then for the nearest hub of pirate activity. You point to Greenrock in Shaula, the biggest pirate world in southern human space, telling them it's one of the most dangerous ones. They then ask if there are any major shipyards in the South, and you point them to Southbound Shipyards, on <planet>, and the Tarazed Corporation, on Wayfarer. After some debate, they decide that <planet>'s proximity to the Coalition makes it a better pick in case they need to return quickly, and settle on that as the destination. "Bringing along enough credits to purchase a few warships, I hope we are," Pyakri comments. They have their <cargo> brought to your ship, and you show them to their bunks.`
```